models/dex/dex_aggregator_trades.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+        alias ='aggregator_trades',
```

Comment:
a more minor comment, which do we prefer?
1. dex_aggregator.trades
2. dex.aggregator_trades

i think the first option is more in line with the majority of the repo, but maybe @soispoke can help me confirm that?

Reply:
Yea right. I just thought we would the other part had to remain dex to stay online with the directory structure.

Reply:
I have been trying to make this `dex_aggregator.trades` work (analogous to the `_blockchain` pattern) with the directory structure, but there must be something I am missing because I can't seem to get it to work. The current diff state of the PR shows my attempt so far.

Reply:
i pushed a change with an edit to dbt project file (which handles the directories and schema names) and edited the table alias -- does this work now?

Reply:
Sure does! Awesome!